.circleci/config.yml

```diff
@@ -101,6 +101,18 @@ commands:
               circleci-agent step halt
             fi
 
+  webpack_build:
```

Comment:
Does CircleCI automatically cache the results from this workflow to be shared across other workflows that depend on it? If not, do we actually gain any benefits from separating this out?

Reply:
Thanks for explaining. Can you provide some of this explanation in a comment here? I think specifically including that this is meant to help developers detect build-specific issues that are unrelated to test failures will help make this much clearer. I had assumed this was being done for a performance reason, and I suspect others that come across this might assume the same.